README.md

```diff
@@ -122,51 +124,38 @@ To run tests, run the following command:
 
 ```sh
   gem install rspec
-  bundle exec rspec
+  rspec spec
 ```
 
-<!--
 ### Deployment
 
-You can deploy this project using:
+You can deploy this project using [render](https://render.com)
 
-<!--
-Example:
-
-```sh
-
-```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Nickson Nowembabazi**
+> Mention all of the collaborators of this project.
 
```

Comment:
**[Optional]**

- [ ] It would be nice if you remove this section description to make your readme look clean and professional 😊 These section descriptions are just instructions for you when you will make your readme file and after making your readme file you are supposed to remove all of them because these are unnecessary for the viewers of your repository 🤔
- [ ] It would be nice if you put your name back because this section is showing the Authors of this repository so this section should have your name and accounts handle too 👍